app/presenters/two_factor_auth_code/phone_delivery_presenter.rb

```diff
@@ -45,6 +45,36 @@ def help_text
       ''
     end
 
+    def troubleshooting_header
+      t('two_factor_authentication.phone_verification.troubleshooting.header')
+    end
+
+    def troubleshooting_options
+      [
+        {
+          url: unconfirmed_phone ? phone_setup_path : login_two_factor_options_path,
+          text: troubleshoot_change_phone_or_option_text,
+        },
+        {
+          url: MarketingSite.help_center_article_url(
+            category: 'get-started',
+            article: 'authentication-options',
+            anchor: 'text-message--phone-call',
```

Comment:
I wonder if it'd be worth creating a custom deep link in the help article to point to? Personally, when testing, I was a bit confused/alarmed that the first thing I read when clicking the option about not receiving my code was some text about how SMS is susceptible to theft and hacking. I might have expected "Didn't receive your one time code?" to be a sub-heading, but I see in the design that it's not. But we could still create a wrapper element with an ID and link to that?

e.g., in https://github.com/18F/identity-site/pull/1028:

```html
<div id="didn-t-receive-your-one-time-code">Didn’t receive your one time code?</div>
```

Reply:
dont!